README.md

```diff
@@ -16,6 +16,17 @@ MongoDB Atlas SQL JDBC driver requires Java 1.8 or higher.
 You can download the precompiled driver (jars) from [Maven Central](https://search.maven.org/artifact/org.mongodb/mongodb-jdbc).  
 Choose `jar` in the Download dropdown for the lean jar (dependencies not included) or `all.jar` for the fat jar (dependencies bundled inside the jar). The name of the driver class is `com.mongodb.jdbc.MongoDriver`.
 
+### Verify Integrity of MongoDB JDBC Driver Packages
+To verify the integrity of the JDBC Driver packages, the artifact signature needs to be verified. To do this, there are two steps:
+1. Import the public key:  
+- You will need to import the key from one of the three following servers: `pgp.mit.edu`, `keys.openpgp.org`, `keyserver.ubuntu.com`.  
```

Comment:
are we sure all 3 of these are correct? mit and ubunutu worked for me but openpgp gives "gpg: keyserver receive failed: No data"

Reply:
This is so odd, Ethan had the same issue. It worked fine for me.